jekyll/_cci2/configuration-reference.md

```diff
@@ -103,6 +103,8 @@ working_directory | N | String | In which directory to run the steps.
 environment | N | Map | A map of environment variable names and values.
 {: class="table table-striped"}
 
+<sup>(1)</sup> exactly one of them should be specified. It is an error to set more than one.
```

Comment:
```suggestion
<sup>(1)</sup> One executor type should be specified per job. If more than one is set you will recieve an error.
```

Reply:
Makes sense, I will update the footnote where I copied this note from as well 👍